app/models/calculator/price_sack.rb

```diff
@@ -24,7 +24,7 @@ def self.description
 
     def compute(object)
       min = preferred_minimal_amount.to_f
-      order_amount = line_items_for(object).map { |x| x.price * x.quantity }.sum
+      order_amount = line_items_for(object).map { |x| x.price * x.quantity }.to_a.sum
```

Comment:
As previous comment above, we should be able to do: `line_items_for(object).sum('price * quantity')` here.